client/tailwind.config.js

```diff
@@ -70,7 +76,11 @@ module.exports = {
       'ButtonText': 'ButtonText',
     },
     fontFamily: {
-      sans: 'var(--w-font-sans)',
+<<<<<<< HEAD
+      sans: 'var(--w-font-sans,-apple-system,BlinkMacSystemFont,"Segoe UI",system-ui,Roboto,"Helvetica Neue",Arial,sans-serif,Apple Color Emoji,"Segoe UI Emoji","Segoe UI Symbol","Noto Color Emoji")',
+=======
+      sans: `var(--w-font-sans,${fallbckFonts[0]})`,
+>>>>>>> 67ba83f82aa5abaa58f51500234ed2d604e44703
```

Comment:
looks like something has gone wrong here with your rebase/merge, can you take a look please